seq2seq/__init__.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+logger = logging.getLogger('seq2seq')
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
```

Comment:
Personally I think displaying the module name is a bit to much.  Maybe remove `%(name)`... Your thought?